docs/source/deployment/ui-custom-base-path.md

```diff
@@ -4,72 +4,35 @@ This document lists the instructions for building an aerie-ui Docker image with
 
 ### Building
 
-1. Clone the [aerie-ui](https://github.com/NASA-AMMOS/aerie-ui) and install dependencies. Note that [Node LTS](https://nodejs.org/) is required (currently 16.17.1).
+1. Clone the [aerie-ui](https://github.com/NASA-AMMOS/aerie-ui) and install dependencies. Note that [Node LTS](https://nodejs.org/) is required (currently 16.18.0).
 
    ```sh
    git clone https://github.com/NASA-AMMOS/aerie-ui.git
    cd aerie-ui
    npm install
    ```
 
-   When you clone aerie-ui the default branch is [develop](https://github.com/NASA-AMMOS/aerie-ui/tree/develop). If you want to build an image from a [specific release](https://github.com/NASA-AMMOS/aerie-ui/releases) you have to checkout the proper tag. For example to checkout [v0.13.0](https://github.com/NASA-AMMOS/aerie-ui/releases/tag/v0.13.0) do:
+   When you clone aerie-ui the default branch is [develop](https://github.com/NASA-AMMOS/aerie-ui/tree/develop). If you want to build an image from a [specific release](https://github.com/NASA-AMMOS/aerie-ui/releases) you have to checkout the proper tag. For example to checkout [v0.13.2](https://github.com/NASA-AMMOS/aerie-ui/releases/tag/v0.13.2) do:
 
    ```sh
-   git checkout tags/v0.13.0 -b v0.13.0
+   git checkout tags/v0.13.2 -b v0.13.2
```

Comment:
Does this need to be updated to keep up with the current version? Asking since I noticed this is a preemptive change for when we release 0.13.2

Reply:
I'm changing this ahead of the release. The link will be active in the docs after the release tomorrow.

Reply:
Yup, I noticed. That's why I was asking if it's something we're going to keep updating as we release new tags.

Reply:
I'd say generally no. I'm just updating it in this case because `0.13.2` and above will not require the step removed from the document.

Reply:
That makes sense 👍